test/__init__.py

```diff
@@ -71,3 +75,53 @@ def _printobj(prefix, obj):
                         _printobj('  -', r)
 
         self.assertEqual(0, len(NativeResource._living))
+
+
+class FileCreator(object):
```

Comment:
style: Just move this helper class to test_s3.py where you're using it

`__init__.py` isn't a great place for stuff because it gets included in EVERYTHING below it. In retrospect, I wish all the `__init__.py` files were empty